packages/mask/src/plugins/Pets/SNSAdaptor/PetDialog.tsx

```diff
@@ -185,7 +185,7 @@ export function PetDialog() {
 
     const renderImg = (address: string) => {
         const imgItem = extraData.filter((i) => isSameAddress(i.address, address))
```

Comment:
```suggestion
        const matched = extraData.find((item) => isSameAddress(item.address, address))
```

see https://mdn.io/array-find